README.md

```diff
@@ -115,23 +118,23 @@ Which one of the following concepts does this best illustrate?
 What is an IIFE, and why might you use it?
 
 **Your answer:**
-
+Immediately invoked function expression - you would use it to call a function immediately after the compiler parses the function
 > ...
 
 ## Question 7
 
 What is the `ui-sref` directive, and how is it used?
 
 **Your answer:**
-
+creates a link in the browser to a view in a SPA
 > ...
```

Comment:
:+1: but more particularly it renders a view associated with a particular **state**.